sinks/kafka/kafka.go

```diff
@@ -277,6 +296,36 @@ func (k *KafkaSpanSink) Start(cl *trace.Client) error {
 // the bytes, messages and interval settings to your tastes!
 func (k *KafkaSpanSink) Ingest(span *ssf.SSFSpan) error {
 
+	// if we've set a sampleTag, we need to determine whether a span should be
+	// sampled.
+	if k.sampleTag != "" {
+		var hashKey uint32
+		sampleTag, exists := span.Tags[k.sampleTag]
```

Comment:
Maybe use `sampleValue` here? It confused me that the you used the same name as the struct's `sampleTag` on first read!

Reply:
Good point!